Update Checkout-sdk.md (#837)
bigcommerce/dev-docs: docs/api-docs/cart-and-checkout/checkout-sdk.md

    -   Amazon
    -   Klarna

> -   You can provide a JavaScript interface for interacting with the web API.

Reviewer: ```suggestion
-   You can use its JavaScript interface for interacting with the web API.

```
